extensions/adobe/experience/customerJourneyManagement/messageprofile.schema.json

```diff
@@ -79,6 +79,11 @@
           },
           "meta:titleId": "messageprofile##https://ns.adobe.com/experience/customerJourneyManagement/pushChannelContext/platform##title##73351",
           "meta:descriptionId": "messageprofile##https://ns.adobe.com/experience/customerJourneyManagement/pushChannelContext/platform##description##33451"
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/smsChannelContext/provider": {
+          "title": "SMS Provider",
+          "type": "string",
+          "description": "SMS provider , e.g. sinch or twilio"
```

Comment:
If this is a pre defined list then we can make this enum.

Reply:
done